subgraphs/badgerdao/src/modules/Deposit.ts

```diff
@@ -92,7 +92,13 @@ export function Deposit(
       constants.BIGINT_ZERO
     );
 
-    depositAmount = totalSupply.minus(vault.outputTokenSupply!);
+    const sharesMinted = totalSupply.minus(vault.outputTokenSupply!);
+
+    depositAmount = vault.outputTokenSupply!.isZero()
```

Comment:
Shouldn't this be done after we set the new vault.inputTokenBalance and vault.outpuTokenSupply?

Reply:
No, this needs to be done with the old values as we need to find the amount deposited in the pool using the shares minted during a deposit transaction.